webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py

```diff
@@ -3,7 +3,7 @@
 from ... import get_viewport_dimensions
 
 import webdriver.bidi.error as error
-from webdriver.bidi.modules.browsing_context import ElementOptions, BoxOptions
+from webdriver.bidi.modules.browsing_context import ElementOptions, BoxOptions, FormatOptions
```

Comment:
While you are at it:

```suggestion
from webdriver.bidi.modules.browsing_context import BoxOptions, ElementOptions, FormatOptions
```